en/components/combo.md

```diff
@@ -6,59 +6,52 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
 
 # Combo
 
-Use the Combo Component to let the user select one or more items from a collection that is displayed upon user interaction in a scrollable list. In case you want to costrain the selection to a single item only, you should use the [Dropdown](dropdown.md) instead. The Combo is visually identical to the [Ignite UI for Angular Combo Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/combo.html)
+Use the Combo Component is used to allow users to select an option from a predefined list of choices or enter their own value. When the user clicks on the drop-down arrow, a drop down list of options is displayed, and they can either select one of the options or type their own value into the text box. The Combo is visually identical to the [Ignite UI for Angular Combo Component](https://www.infragistics.com/products/ignite-ui-angular/angular/components/combo.html)
 
 ## Combo Demo
 
-The Combo has two pieces: an input containing the current selection and a dropdown that is shown in order to select one or more items.
+The Combo has two pieces: an input containing the current selection and a drop down that is shown in order to select one or more items.
 
 <img class="responsive-img" src="../images/combo_demo.png" srcset="../images/combo_demo@2x.png 2x" />
 
-## Input Type
+## Combo Input Type
 
-The Combo input provides selection between a line type for a more airy style, border type for a more structured perception on solid color backgrounds, and a boxed type that is most appropriate when placed on top of a vivid image to improve readability.
+The Combo provides selection between a border type for a more structured perception on solid color backgrounds, and a boxed type that is most appropriate when placed on top of a vivid image to improve readability.
 
-<img class="responsive-img" src="../images/combo_demo.png" srcset="../images/combo_demo@2x.png 2x" />
-`line`
 
 <img class="responsive-img" src="../images/combo_border.png" srcset="../images/combo_border@2x.png 2x" />
```

Comment:
The types are swapped. On the border type it says box and on the box type it says border